src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -733,10 +749,37 @@ internal async Task SearchPackagesAndRefreshUpdateCountAsync(string searchText,
                 loadContext.CachedPackages = Model.CachedUpdates;
             }
 
-            var packageFeed = await CreatePackageFeedAsync(loadContext, _topPanel.Filter, _uiLogger);
+            // only make recommendations when
+            //   the single source repository is nuget.org,
+            //   the package manager was opened for a project, not a solution,
+            //   this is the Browse tab,
+            //   and the search text is an empty string
+            if (loadContext.SourceRepositories.Count() == 1
+                && loadContext.SourceRepositories.First().PackageSource.Source.ToLower() == "https://api.nuget.org/v3/index.json"
```

Comment:
There should be a utility that already does this. 
Search for "IsNuGetOrg" or something like that.

Reply:
I changed it to TelemetryUtility.IsNuGetOrg. But should we be using TelemetryUtility for this? I couldn't find any other method that does this.